### DISCUSSION TITLE
Sui dati dei comuni

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/informa-covid19#20] [Generale]
Buongiorno a tuttə,
apro una discussione fuori dalle issue, per aprire un discorso più largo, da approfondire poi con issue verticali dedicate.

Inizio con delle domande:

- i dati del comune di Regalbuto, sono già nel formato di output che si sta immaginando per il progetto? 
- per ogni comune ci sarà un solo "file"?

Grazie

**Comment by dennisangemi** [comment]
Buongiorno!
- Se per "formato di output" intendi lo schema dati proposto da @gabacode, sì. La struttura dei file di output di Regalbuto, Catenanuova, Serradifalco, + i 5 comuni di gaba è uguale. L'unica cosa che rimane da modificare è la colonna "distretto" che diventerà "sigla_provincia".
- Attualmente sì. Pensi sia necessario altro?

**Reply by aborruso** [reply]
> * Se per "formato di output" intendi lo schema dati proposto da @gabacode, sì. La struttura dei file di output di Regalbuto, Catenanuova, Serradifalco, + i 5 comuni di gaba è uguale. L'unica cosa che rimane da modificare è la colonna "distretto" che diventerà "sigla_provincia".

Ok @dennisangemi così posso iniziare a lavorare su #13 

> * Attualmente sì. Pensi sia necessario altro?

Non servono altro, ma prima di iniziare a lavorarci devo essere sicuro di tutto

**Comment by dennisangemi** [comment]
[Questo file](https://docs.google.com/spreadsheets/d/1eaKiVQhifRZpxqLh2vUCk4F7vm3nNP9nz67GrzArqtI/edit?usp=sharing) potrebbe essere utile per capire quali comuni hanno lo stesso schema dati e quali no

**Reply by aborruso** [reply]
> [Questo file](https://docs.google.com/spreadsheets/d/1eaKiVQhifRZpxqLh2vUCk4F7vm3nNP9nz67GrzArqtI/edit?usp=sharing) potrebbe essere utile per capire quali comuni hanno lo stesso schema dati e quali no

@dennisangemi  mi dici uno solo, che è il più vicino al formato di output pensato? Non mi fare una lista, scrivimi per favore ad esempio "xxxxx".

Grazie

**Reply by dennisangemi** [reply]
Regalbuto ;)

**Reply by gabacode** [reply]
@dennisangemi mettiamo il codice ISTAT anche nel nome file?
(es. ```dpc-covid19-ita-en-86016.csv``` vs ```dpc-covid19-ita-en-regalbuto -  .csv```)

@aborruso il padding 0 conviene metterlo anche in questi files? (es. ```dpc-covid19-ita-en-086016.csv```) ?

**Reply by aborruso** [reply]
Gabriele direi sì a entrambe

Grazie

**Reply by dennisangemi** [reply]
Il csv generato dal gsheet prende il nome del foglio quindi per rinominare il file bisognerebbe fare degli step successivi che metterebbero in crisi il/la sindaco/a o chi per lui/lei. Per quanto riguarda il nome del file che si caricherà (?) su github @aborruso farà qualche magheggio

**Reply by aborruso** [reply]
Ok, generata prima versione dello script, che ogni giorno alle 18:30 circa, aggiornerà qui nel repo i dati dai comuni che hanno aderito

https://github.com/opendatasicilia/informa-covid19/blob/main/script/informacovid/README.md

**Comment by aborruso** [comment]
Sì, per il mio lavoro serve l'URL del file (qualunque esso sia) e codice
ISTAT del comune.
Poi nel nostro repo sarà salvato con un nome normalizzato

______
Scusami per la brevità, ti sto scrivendo dal cellulare.

website: https://medium.com/tantotanto
38° 7' 48" N, 13° 21' 9" E EPSG:4326

Il sab 1 mag 2021, 14:12 Dennis Angemi ***@***.***> ha
scritto:

**Reply by dennisangemi** [reply]
Grazie mille
